aspnetcore/log-mon/metrics/metrics.md

```diff
@@ -262,6 +263,51 @@ The proceeding test:
 * Makes an HTTP request to the web app.
 * Asserts the test using results from the metrics collector.
 
+:::moniker range=">= aspnetcore-10.0"
+
+## ASP.NET Core Identity metrics
+
+ASP.NET Core Identity observability has been improved in .NET 10 with metrics. These metrics help you monitor user management activities and authentication processes.
+
+The metrics are in the `Microsoft.AspNetCore.Identity` meter and include:
```

Comment:
I don't think the style manul permits colons before headings, or at least it doesn't state they're acceptable, nor does Chicago. Recommend something more like ...

```suggestion
The metrics are in the `Microsoft.AspNetCore.Identity` meter and are described by the following sections.
```